tests/UI/pages/BO/orders/index.js

```diff
@@ -277,6 +277,17 @@ class Order extends BOBasePage {
     );
   }
 
+  /**
+   * Get order total price
+   * @param page
+   * @param row
+   * @return {number}
+   */
+  async getOrderTTCPrice(page, row) {
```

Comment:
I'm pretty sure "TTC" is a french acronym :(

Reply:
I think it's ATI (All Taxes Included)

Reply:
It should be done for all function in this tests (and not breaking others)

Reply:
That's a problem ?

Reply:
no it all good now for this PR